errorHandler/api.errorHandler.js

```diff
@@ -0,0 +1,18 @@
+
+exports.handleCustomErrors = ((err, req, res, next)=>{
+    if(err.status === 404 && err.msg === '404 Not Found'){
+        res.status(err.status).send(err)
+    }
+    next(err)
+})
```

Comment:
Fine to do it like this, you can also generalize this out a bit more - if you just check whether the error has a status & a msg, you can always just send off the status & message -> no matter if it's a 404 or a different custom handled error!